apps/prairielearn/elements/pl-external-grader-results/pl-external-grader-results.py

```diff
@@ -27,10 +30,11 @@
     "#5ffdff",
     "#feffff",
 )
-conv = Ansi2HTMLConverter(inline=True, scheme="iterm")
+
+conv: Ansi2HTMLConverter = Ansi2HTMLConverter(inline=True, scheme="iterm")
```

Comment:
For my own knowledge: is the explicit type annotation necessary? I thought the type checker could infer types.

Reply:
I think it's unnecessary.